src/profile-logic/sanitize.js

```diff
@@ -221,6 +222,17 @@ function sanitizeThreadPII(
         stringArray[stringIndex] = stringArray[stringIndex].replace(/:.*/, '');
       }
 
+      // Remove the all OS paths from FileIO markers if user wants to remove them.
+      if (
+        PIIToBeRemoved.shouldRemoveUrls &&
```

Comment:
I think that we should expose the change to the UI, by simply changing `Include resource URLs` to `Include resource URLs and paths`. What do you think?